README.md

```diff
@@ -20,7 +20,6 @@ The Casper MainNet is live.
 
 ## Get Started with Smart Contracts
 - [Writing Smart Contracts](https://docs.casperlabs.io/en/latest/dapp-dev-guide/index.html)
-- [Rust Smart Contract SDK](https://crates.io/crates/cargo-casper)
```

Comment:
I think it's OK to leave this link here - it should remain valid.

Reply:
fixed.